### PR TITLE
Adding fix for Serverless ingress hack

### PR DESCRIPTION
**Pull request by matzew**

### CODE REVIEW

File: deploy/resources/networkpolicies.yaml [hunk 1]
```diff
@@ -26,3 +26,17 @@ spec:
   policyTypes:
   - Ingress
   - Egress
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: knative-eventing-cluster-local-gateway-hack
+  namespace: knative-serving-ingress
```
Comment (matzew) [inline deploy/resources/networkpolicies.yaml:39]: fancy name, and explicit namespace

File: deploy/resources/networkpolicies.yaml [hunk 2]
```diff
@@ -26,3 +26,17 @@ spec:
   policyTypes:
   - Ingress
   - Egress
+apiVersion: networking.k8s.io/v1
```
Comment (maschmid) [inline deploy/resources/networkpolicies.yaml:35]: Missing --- ?

**Review by maschmid — CHANGES_REQUESTED** [review]
Seems the yaml separator is missing at the beginning...

File: deploy/resources/networkpolicies.yaml (hunk 2 above)
Comment (matzew) [reply deploy/resources/networkpolicies.yaml:35]: yup

**Review by maschmid — APPROVED** [review]
lgtm

**Pull request merged**